src/backend/blockchain/tests/functional/test_routes.py

```diff
@@ -114,57 +139,66 @@ def test_add_wallet(test_client):
     response = test_client.post(url, data=json.dumps(data), headers=headers)
 
     assert response.status_code == 201
-    assert response.json['success'] == True
+    assert response.json['success']
```

Comment:
lol what are we doing over here

Reply:
That was probably from my original blockchain tests hahaha whoops